src/Camera.cpp

```
@@ -0,0 +1,66 @@
+		setCameraStatus(false);
+		return cv::Mat();
+	}
+
+	return frame;
+}
+
+void Camera::setCameraStatus(bool status) {
+	updateDeviceStatus(0, status);	// Camera is index 0
+}
+
+bool Camera::getCameraStatus() const {
```

Cppcheck (2.10-2)

[style] The function 'getCameraStatus' is never used.

(unusedFunction)